src/components/radio-group/radio-group.ts

```diff
@@ -90,11 +92,29 @@ export default class IgcRadioGroupComponent extends LitElement {
     return this._value;
   }
 
+  private _observerCallback() {
+    const radios = Array.from(this._radios as unknown as Element[]).filter(
+      (el) => el.tagName.toLowerCase() === 'igc-radio'
+    ) as IgcRadioComponent[];
```

Comment:
```suggestion
    const radios = Array.from(this._radios)
```

Isn't `this._radios` already typed as an `NodeListOf<IgcRadioComponent>`?